warriorsOfTheDarkGods.cat

```diff
@@ -4206,7 +4206,7 @@ Models with this Favour gain +1 Resilience. If a model with this Favour Declares
       <rules>
         <rule id="e227-4787-adce-c84e" name="Favour of Vanadra, Goddess of Wrath" hidden="false">
           <description>Attack Attribute - Melee
-The model part gains Lightning Reflexes and +1 Agility. Close Combat Attacks allocated towards the model gain +1 to hit. These effects are only applied in the first Round of each Combat.</description>
+            The model part gains Lightning Reflexes and +1 Agility. Close Combat Attacks allocated towards the model gain +1 to hit. These effects are only applied in the first Round of each Combat.</description>
```

Comment:
Why are there unnecessary blanks before this line?

Reply:
I'm updating the files manually and the lack of space makes it more difficult for me to collapse sections I don't need. It doesn't affect the printed output in BS (I checked).